examples/next/pages/ui/components/authenticator/custom-slots/index.page.tsx

```diff
@@ -15,6 +15,7 @@ import {
 } from '@aws-amplify/ui-react';
 
 import awsExports from './aws-exports';
+import { AuthFormFields } from '@aws-amplify/ui';
```

Comment:
Sorry missed this on the earlier revisions but can we have this example without the type? Would prefer we don't guide users to import from our internal packages

Reply:
+1, also means JS users can't just copy/past our examples.